<launch>
    <arg name="publish_gesture_topic" default="/gesture/gesture_name"/>
    <arg name="control_topic" default="/mobile_robot/cmd_vel"/>

    <node name="sign_to_controller" pkg="ros_hand_gesture_recognition" type="sign_to_controller.py" output="screen">
        <param name="publish_gesture_topic" type="string" value="$(arg publish_gesture_topic)" />
        <param name="control_topic" type="string" value="$(arg control_topic)" />
    </node>
</launch>
